extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -323,6 +359,10 @@ trait LineageParser {
         val tableName = p.name
         joinRelationColumnLineage(parentColumnsLineage, p.output, Seq(tableName))
 
+      case p: DataSourceV2Relation =>
```

Comment:
we are pasring optimized plan, it seems we can never get DataSourceV2Relation ?  it would be always converted DataSourceV2ScanRelation

Reply:
This scenario occurs when creating a view using the v2 table, because it is not going to read it, here corresponding to the `relation` is `DatasourceV2Relation`.

Reply:
I see. can we add some comments here, it can help recall in future